tests/manage/z_cluster/nodes/test_simultaneous_drain_of_two_ocs_nodes.py

```diff
@@ -0,0 +1,141 @@
+import logging
+import pytest
+from ocs_ci.framework.testlib import (
+    tier4, tier4c, ManageTest,
+    aws_platform_required,
+    ipi_deployment_required, ignore_leftovers
+)
+from ocs_ci.ocs import machine, constants
+from tests.sanity_helpers import Sanity
+from ocs_ci.ocs.node import (
+    drain_nodes, get_osd_running_nodes,
+    get_node_objs, remove_nodes, add_new_node_and_label_it
+)
+from tests.helpers import wait_for_resource_state, label_worker_node
+from ocs_ci.ocs.resources import pod
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(autouse=True)
+def Delete_unscheduled_nodes_teardown(request):
+    """
+    Remove unscheduled nodes from the OCS cluster
+    """
+    def finalizer():
+        unscheduled_nodes = [
+            n for n in get_node_objs() if n.ocp.get_resource_status(
+                n.name
+            ) == constants.NODE_READY_SCHEDULING_DISABLED
+        ]
+        remove_nodes(unscheduled_nodes)
+    request.addfinalizer(finalizer)
+
+
+@tier4
+@tier4c
+@ignore_leftovers
+@aws_platform_required
```

Comment:
Is aws limitation still required? same for ipi as well?

Reply:
In this test we are adding nodes by increasing machine set. So, this is specific to aws for now.
I will add UPI, once the functions for add_node are ready.